datajoint/autopopulate.py

```diff
@@ -235,17 +236,16 @@ def handler(signum, frame):
             del self.connection._conn.ctx  # SSLContext is not pickleable
             with mp.Pool(
                 processes, _initialize_populate, (self, jobs, populate_kwargs)
-            ) as pool:
-                if display_progress:
-                    with tqdm(desc="Processes: ", total=nkeys) as pbar:
-                        for error in pool.imap(_call_populate1, keys, chunksize=1):
-                            if error is not None:
-                                error_list.append(error)
-                            pbar.update()
-                else:
-                    for error in pool.imap(_call_populate1, keys):
-                        if error is not None:
-                            error_list.append(error)
+            ) as pool, (
+                tqdm(desc="Processes: ", total=nkeys)
+                if display_progress
+                else contextlib.nullcontext()
+            ) as progress_bar:
+                for error in pool.imap(_call_populate1, keys, chunksize=1):
+                    if error is not None:
+                        error_list.append(error)
+                    if display_progress:
+                        progress_bar.update()
```

Comment:
This implementation has only one implementation of the inner loop that appends to the list. Previous implementation had to repeat the inner loop twice.

Reply:
Gotcha. Was just confirming if this should be reported in the `CHANGELOG`. Looks like it is just a maintenance update so we don't need to report it since it won't affect UX.